specs/Checkbox.md

```diff
@@ -173,7 +147,25 @@ Removing the following two props because the ARIA spec dictates role='checkbox'
 
 Props being removed:
 
-ariaPoisitionInSet and ariaSetSize - when writing parent component, user should set these on the checkbox.
+ariaPoisitionInSet and ariaSetSize - when writing parent component, user should set these on the checkbox. ARIA spec dictates role='checkbox' doesn't need aria-posinset and aria-setsize. These are only valid for role='option' which is only in the case the checkbox is a part of a listbox, which is not something we need to account for in the base component API. If the user does need to provide these two props, slotProps could be used to apply additional props to any slot. 
+
+Note: rtl, styles, and theme come from compose or the ThemeProvider. And name has been added to support checkbox in form scenarios.
+
+### Work items and tracking progress on conversion from the current Fluent (Stardust) checkbox to the new Fluent checkbox we're building with this spec: 
+- [ ] Remove `animation` prop.
+- [ ] Remove `accessibility` prop.
```

Comment:
as discussed on the sync last week, we should not remove accessibility prop. We can have a separate discsussion about a replacement for it

Reply:
Ah I wasn't aware of this - what is this separate plan/is it documented anywhere?